src/Splunk.Logging.Common/TcpSocketWriter.cs

```diff
@@ -95,17 +95,17 @@ public class TcpSocketWriter : IDisposable
             else
             {
                 this.tryOpenSocket = (h, p) =>
+                {
```

Comment:
You just removed an unnecessary indent here, right? Making sure I didn't miss anything.